content/en/blog/news/2022-introduction-to-sbom.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Introduction to Software Bill Of Materials"
+date: 2022-07-24
+draft: false
+weight: 300
+description: >
+  Here we discuss the concepts of SBOM, how it can help in securing your software.
+categories: [blog]
+keywords: [Community, GSoC, 2022, supply chain security, BOM, SBOM]
+slug: "intro-to-sbom"
+aliases: []
+author: Osama Magdy
+---
+
+## Introduction
+
+Before going through Software Bill Of Materials, we need to set the ground for a rising concern in the software industry which is Software Supply Chain Security.
+Like traditional industries, deploying a piece of a software artifact goes through multiple stages composed of collecting source code components, libraries, tools, and processes used in those stages.
+
+![](/images/sbom-guide/supply-chain.png)
+
+Fig. 1 https://blog.convisoappsec.com/en/is-your-software-supply-chain-secure/
+
+A supply chain attack can occur along the chain from submitting unauthorized malicious code in your source, injecting harmful dependencies with backdoor ways to your authentication system, and even replacing packages after being built with other compromised artifacts.
```

Comment:
may be replace `injecting harmful dependencies with backdoor ways to your authentication system` with `unauthorized injection of harmful dependencies`

Reply:
Done